en/micro-integrator/docs/references/connectors/db-event-inbound-endpoint/db-event-inbound-endpoint-example.md

```diff
@@ -0,0 +1,139 @@
+# DB Event inbound endpoint example
+
+Data is the most valuable asset in any business. Almost every cooperate system has a on-remise or cloud based data storage facilities. When the individual systems in a particular business are integrated together, sometimes they are coupled via database systems. For an example, one system can write data while another systems reads and process them. On such instances, the systems may want to know if there any changes to the data being performed by external parties or systems. 
+
+Hence, for an enterprise integration platform it is a useful feature to be able to generate events based on the data changes. **DB Event Inbound Endpoint** is the DB event listener for `WSO2 Enterprise Integrator`. You can configure it with any popular Database systems such as `MySQL` and `Oracle` etc. 
+
+Following are the main features of the event generator. 
+
+1. Trigger an event with the data of table row when a new record is added or updated. Optionally, delete the row associated with the event after triggering the event
+2. Trigger an event when a boolean field is flipped in a particular table row. 
+
+## What you'll build
+
+In this example let us see how to configure `DB-event Inbound Endpoint` so that it can listen to data changes done to a `MySQL` table. Out of the features mentioned above feature no:1 is used here. Please refer to [reference guide](db-event-inbound-endpoint-config.md) if you need to use other features. 
+
+In the system of an enterprise a relational database table is used to store customer information. Customers' information is added by an external system to the database which is not in enterprise's control. As soon as a new customer is inserted, the system need to pick up and process its data. `WSO2 EI` is used here to listen to DB changes and invoke the relevant processes. It can invoke backend APIs or place data into a message bus after required data transformations. However, for simplicity of this example, we will just log the message. You can extend the sample as required using WSO2 EI mediators. 
```

Comment:
```suggestion
In an enterprise system, a relational database table is used to store customer information. The customer's information is added by an external system to the database, which is not in the enterprise's control. As soon as a new customer is inserted, the system need to pick up and process its data. `WSO2 EI` is used here to listen to DB changes and invoke the relevant processes. It can invoke backend APIs or place data into a message bus after required data transformations. However, for simplicity of this example, we will just log the message. You can extend the sample as required using WSO2 EI mediators. 
```

Reply:
Updated.